pdns/dbdnsseckeeper.cc

```diff
@@ -450,7 +450,7 @@ bool DNSSECKeeper::getPreRRSIGs(UeberBackend& db, const DNSName& signer, const D
                         // cerr<<"Got it"<<endl;
                         if (wildcardname.countLabels())
                                 rr.qname = qname;
-                        rr.d_place = (DNSResourceRecord::Place)signPlace;
+                        rr.d_place = static_cast<DNSResourceRecord::Place>(signPlace);
```

Comment:
this should be DNSRecord::Place now, or signPlace should be of the right type anyhow. Is only a warning and not a real issue so we could fix this after the merge.